package.json

```diff
@@ -47,121 +52,64 @@
     "gulp-connect": "3.2.2",
-    "gulp-csso": "2.0.0",
-    "gulp-if": "2.0.0",
-    "gulp-imagemin": "2.4.0",
-    "gulp-less": "3.0.5",
-    "gulp-replace": "0.5.4",
-    "gulp-sourcemaps": "1.6.0",
-    "gulp-uglify": "1.5.3",
-    "gulp-util": "3.0.7",
+    "html-loader": "0.4.3",
+    "html-webpack-plugin": "2.16.0",
+    "image-webpack-loader": "1.7.0",
     "jasmine-reporters": "2.1.1",
     "jest-cli": "11.0.2",
     "jest-webpack-alias": "2.2.0",
     "json-loader": "0.5.4",
+    "less": "2.6.1",
+    "less-loader": "2.2.3",
     "localStorage": "1.0.3",
     "marked": "0.3.5",
-    "mkdirp": "0.5.1",
     "node-libs-browser": "1.0.0",
-    "run-sequence": "1.1.5",
+    "postcss-loader": "0.8.2",
+    "react-hot-loader": "1.3.0",
     "source-map-loader": "0.1.5",
+    "string-replace-webpack-plugin": "0.0.3",
+    "style-loader": "0.13.1",
     "transform-loader": "0.2.3",
     "webpack": "1.13.0",
+    "webpack-dev-server": "1.14.1",
     "webpack-notifier": "1.3.0"
   },
   "scripts": {
-    "clean": "rm -rf build",
-    "prebuild-ee": "./scripts/validate-tests",
-    "build-ee": "npm run clean && npm test && NODE_ENV='production' ./node_modules/.bin/gulp dist",
-    "dist": "npm run build-ee",
-    "prebuild-ce": "./scripts/validate-tests",
-    "build-ce": "npm run clean && npm test && NODE_ENV='production' ./node_modules/.bin/gulp dist",
-    "serve": "NODE_ENV='development' ./node_modules/.bin/gulp serve",
-    "serve-notify": "export NOTIFY=true && npm run serve",
-    "testing": "NODE_ENV='testing' ./node_modules/.bin/gulp testing",
+    "clean": "rm -rf dist",
+    "build": "npm run clean && npm run lint && npm test && npm run dist",
+    "build-with-external-plugins": "npm run clean && npm run lint && npm run lint-plugins && npm test && npm run dist",
     "lint": "./node_modules/.bin/eslint '**/*.js' --quiet",
-    "livereload": "NODE_ENV='development' ./node_modules/.bin/gulp livereload",
-    "livereload-notify": "export NOTIFY=true && npm run livereload",
-    "test": "./node_modules/.bin/gulp copy:external-plugins && npm run lint && jest --no-cache"
+    "lint-plugins": "PLUGINS=$npm_package_config_external_plugins; ./node_modules/.bin/eslint \"$PLUGINS/**/*.js\" --quiet",
+    "prebuild": "./scripts/validate-tests",
+    "serve": "NODE_ENV='development' npm run dist && ./node_modules/.bin/gulp serve",
+    "serve-notify": "export NOTIFY=true && npm run serve",
+    "scaffold": "./node_modules/.bin/gulp ensureConfig && ./node_modules/.bin/gulp ensureDevProxy",
+    "start": "PORT=$npm_package_config_port; webpack-dev-server --config ./webpack/webpack.dev.babel.js --hot --progress --colors --content-base ./build --port $PORT",
+    "test": "node jest/gen-config.js && NODE_PATH=node_modules jest --config=jest/config.json --no-cache",
+    "testing": "NODE_ENV='testing' npm run dist && ./node_modules/.bin/gulp serve",
+    "dist": "NODE_ENV='production' webpack --config ./webpack/webpack.production.babel.js"
```

Comment:
I like the idea of renaming `dist` to something else.
What if this became `build-application`?